JS/Lesson34/goksel/recipe-app/src/AppRouter.jsx

```diff
@@ -0,0 +1,40 @@
+import { Navigate, Route, Routes } from "react-router-dom"
+import { NewRecipe } from "./modules/recipes/NewRecipe/NewRecipeForm"
+import { RegistrationForm } from "./modules/user/RegistrationForm";
+import { Settings } from "./modules/user/Settings";
+import { RecipeDetail } from "./modules/recipes/RecipeDetail";
+import { SearchBar } from "./modules/recipes/SearchBar";
+import { RecipeList } from "./modules/recipes/RecipeList";
+
+export const AppRouter = ({ recipes, setRecipes, setIsSignedIn, isSignedIn }) => {
+    return (
+        <Routes>
+            <Route path="user">
+                <Route path="register" element={<RegistrationForm setIsSignedIn={setIsSignedIn} isSignedIn={isSignedIn} />} />
```

Comment:
Add the registration page to the Navbar, it's not accessible from it right now.

Reply:
It's accessible now. I used registration form as sign in thats why ı did not add it to the navbar. Actually ı was know that was not right but ı was wondering your comment so it can be more clear for me.